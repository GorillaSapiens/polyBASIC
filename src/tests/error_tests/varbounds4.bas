a(6,6) := 5
a(5) := 6
