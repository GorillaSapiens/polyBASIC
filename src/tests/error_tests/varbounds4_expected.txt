ERROR: @2:12, ARRAY INDEX SIZE MISMATCH ❮a❯ DIM2 @1
